examples/tts/conf/talknet-durs.yaml

```diff
@@ -30,8 +30,9 @@ model:
         punct: true
         spaces: true
         stresses: false
+        add_blank_at: last
     dataloader_params:
-      drop_last: false
+      drop_last: true
```

Comment:
This doesn't really do what you want in lightning. So I would advise you against adding this.

Reply:
Ok.